cherrypicks_test.go

```diff
@@ -185,3 +185,32 @@ Hello :smile_cat: This PR contains changelog entries. Please, verify the need of
 		})
 	}
 }
+
+func TestCherryTargetBranches(t *testing.T) {
+	tests := map[string]struct {
+		input    string
+		expected []string
+	}{
+		"Success": {
+			input: `
+cherry pick to:
+    * 2.6.x
+    * 2.5.x
+    * 2.4.x
```

Comment:
Ah, now I understand the idea behind the regex!

Okey, but I was expecting something simpler: as I user I would have written "cherry pick to: 2.5.x and 2.4.x" and expect it to work, or "cherry pick to 2.6.x, 2.5.x". --> In other words, not being strict in having one version per line.

But I see that this makes the code simpler, so we can start as-is, I have no strong opinion.

Reply:
Yes, I see what you mean, and I see that others have done so too, but let's keep it simple for now :)

If we get all the other parts to work, this should not be too hard to rework later on